spec/database_admin_spec.rb

```diff
@@ -1736,38 +1736,32 @@ def confirm_and_execute
     end
 
     describe "#ask_custom_file_options" do
```

Comment:
Looking back at `describe "#ask_custom_file_options" do` in the last PR, you only added this for the current context this is under (I think just "DB Dump").

Can you add specs for the other two?  (you should be able to grep for the above to find the other places where this is at).